src/test/java/Search.java

```diff
@@ -0,0 +1,105 @@
+import InternSearch.pages.LoginPageSearch;
+import InternSearch.pages.SearchPageSearch;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Search extends BaseTest {
+    String songName = "Dark Days";
+    //use all cases- exact spelling of song name, ingnore trailing and hading white space
+    String [] list = {"Dark Days"," Dark Days ","Dark Days "};
+
+    String [] caseSenList = {"darks day", "darKs DAy"};
```

Comment:
POM pattern uses page objects that represent a web page (or a form within a page, if applicable) to separate test code (validations and test flow logic, for example) from page specific code. It does so by making all actions that can be performed on a page available as methods of the page object representing that page.

For example:

**SearchPageSearch = SearchTest**

![image](https://user-images.githubusercontent.com/104761482/209535673-1468463d-6860-46e9-9ff4-81f371a44541.png)

Reply:
Yes, I moved all properties back to Search page where it belongs.